lifeguard/validations.py

```diff
@@ -143,7 +144,10 @@ def wrapped(*args, **kwargs):
                     extra={"traceback": traceback.format_exc()},
                 )
 
-        VALIDATIONS[decorated.__name__] = {
+        __fname__ = "__" + decorated.__name__ + str(uuid.uuid1()).split('-')[-1]
+
+        VALIDATIONS[__fname__] = {
+            "__name__": decorated.__name__,
```

Comment:
@rosineygp, I understand your worry but I think it isn't better solution to resolve. Actually I think this isn't a real problem.

Use the same function name to validate different things should be rethought.

But let's say it really is a problem a better solution is use complete path to function (module.function_name). And functions with the same name in the same module is a wrong anyway.

The use of an uuid for a dict's key in this case it would be better to use an array.

Besides these points, if we called load_validations more than once we would add more times the same validation, thinking that an application that will work with plugins would not be a very safe behavior. It would be more trouble than override a validation because the same name was used. Using the full path to function would also solve the problem.